src/main/kotlin/com/vanniktech/dependency/graph/generator/ProjectDependencyGraphGenerator.kt

```diff
@@ -71,25 +72,21 @@ internal class ProjectDependencyGraphGenerator(
   }
 
   private fun addDependencies(dependencies: MutableList<ProjectDependencyContainer>, graph: MutableGraph) {
-    val rootNodes = graph.rootNodes().filter { it.links().isEmpty() }
+    val rootNodes = graph.rootNodes().filterNotNull().filter { it.links().isEmpty() }
     dependencies
       .filterNot { (from, to, _) -> from == to }
-      .distinctBy { (from, to, _) -> from to to }
```

Comment:
No longer needed?

Reply:
The reason behind this is explained in https://github.com/vanniktech/gradle-dependency-graph-generator-plugin/pull/172/commits/c58e991af7d8d98529c82bbad7ace52e5dcbeab6

> Keep track of all potential configurations
> 
> Filtering in such way:
> - Prevents to display alternative configurations
> - Introduces a bug where only the first (undefined behavior) configuration would be taken into > account and could lead to false result.